pkgs/models/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, recurseIntoAttrs, stdenvNoCC, fetchurl, makeWrapper, sticker }:
+
+let
+  stickerModel = import ./model.nix;
+
+  # Embeddings fetcher, uses the sticker-models repository.
+  fetchEmbeddings = { name, sha256 }: {
+    embeds = fetchurl {
+      inherit sha256;
+      url = "https://github.com/stickeritis/sticker-models/releases/download/${name}/${name}.fifu";
+    };
+
+    filename = "${name}.fifu";
+  };
+
+  deWordEmbeds = fetchEmbeddings {
+    name = "de-structgram-20190426-opq";
+    sha256 = "0b75bpsrfxh173ssa91pql3xmvd7x9f2qwc6rv27jj6pxlhayfql";
+  };
+in lib.mapAttrs (_: value: recurseIntoAttrs value) {
+  de-pos-ud = stickerModel {
```

Comment:
`stickerModel` is defined through `models/model.nix` (line 4 of this file), below are the attributes (correct terms?) to get the german UD POS tagging model with above defined embeddings.

Reply:
Correct term :).